packages/drom_lib/drom_lib.0.8.0/opam

```diff
@@ -0,0 +1,63 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+synopsis:
+  "The drom tool is a wrapper over opam/dune in an attempt to provide a cargo-like user experience"
+description: """\
+The drom tool is a wrapper over opam/dune in an attempt to provide a cargo-like
+user experience. It can be used to create full OCaml projects with
+sphinx and odoc documentation. It has specific knowledge of Github and
+will generate files for Github Actions CI and Github pages.
+"""
+authors: [
+  "Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"
+  "Léo Andrès <leo.andres@ocamlpro.com>"
+]
+maintainer: [
+  "Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"
+  "Léo Andrès <leo.andres@ocamlpro.com>"
+]
+homepage: "https://ocamlpro.github.io/drom"
+doc: "https://ocamlpro.github.io/drom/sphinx"
+bug-reports: "https://github.com/ocamlpro/drom/issues"
+dev-repo: "git+https://github.com/ocamlpro/drom.git"
+tags: "org:ocamlpro"
+build: [
+  ["dune" "subst"] {dev}
+  ["sh" "-c" "./scripts/before.sh build '%{name}%'"]
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["sh" "-c" "./scripts/after.sh build '%{name}%'"]
```

Comment:
This does nothing

Reply:
Yes, but these `opam` files are automatically generated, so this line is always put in case there is a post-install script to run after installation (the idea is that such a script can be run both by `opam install`, but also by `make install` or any other installation tool).